src/helpers/common.ts

```diff
@@ -35,3 +35,19 @@ export const pickTextColorBasedOnBgColorSimple = (bgColor) => {
 
   return r * 0.299 + g * 0.587 + b * 0.114 > 186 ? 'rgba(0, 0, 0, 0.5)' : '#ffffff';
 };
+
+export const getBadgeColor = (points, minQuestionPoint, maxQuestionPoint) => {
+  const questionPointsRange = maxQuestionPoint - minQuestionPoint;
+
+  if (parseInt(points) > minQuestionPoint + questionPointsRange * 0.8) {
```

Comment:
Always pass radix when using parseInt for safety.

Reply:
Maybe we should we `>=` in this first if logic so if all questions have same points (range = 0, min = max, points = min + 0.8 * range) then all the badges should be green

Reply:
if you want all badge green then it should be `>` only

Reply:
Ohh. I thought the first one is green. My bad.